display_list/dl_builder.cc

```diff
@@ -1302,6 +1302,48 @@ void DisplayListBuilder::DrawTextBlob(const sk_sp<SkTextBlob>& blob,
   SetAttributesFromPaint(paint, DisplayListOpFlags::kDrawTextBlobFlags);
   drawTextBlob(blob, x, y);
 }
+
+void DisplayListBuilder::drawTextFrame(
```

Comment:
Does this mean that `drawTextBlob` will go unused in the Impeller backend?

If so, would you want to (ifdef'd?) make sure it's not used, i.e. `UNIMPLEMENTED`, similar to the dispatcher?

Reply:
I'm not sure if we use DisplayListBuilder on Skia ever. FYI @flar

Reply:
Yes, we do. That was its original purpose and is iteratively becoming its only purpose.